docs/fsharp/language-reference/value-options.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Value Options (F#)
+description: Learn about the F# Value Option type, which is a struct version of the Option type.
+ms.date: 06/16/2018
+---
+
+# Value Options
+
+The Value Option type in F# is generally used when the following two circumstances hold:
+
+1. A scenario is appropriate for an [F# Option](options.md).
+2. Using a struct provides a performance benefit in your scenario.
+
+It's important to note that not all performance-sensitive scenarios are "solved" by using structs, and you must consider the additional cost of copying when using them instead of reference types. However, in large F# programs it is common to have many Optional types instantiated and flowed through hot paths, where structs can sometimes yield better overall performance over the lifetime of a program.
```

Comment:
nit: comma after "large F# programs"
"Optional types instantiated and flowed through hot paths" -->
- Does Optional have to be capitalized?
- awkward. Perhaps, "However, large F# programs commonly instantiate many optional types that flow through hot paths, because structs can sometimes yield better overall performance over the lifetime of a program."?